redhat-operators/businessautomation-operator/businessautomation-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: businessautomation-operator.v1.0.0
+  namespace: placeholder
+  annotations:
+    categories: kieapp, rhpam, rhdm, kie, cloud, pam, bpm, process, case, business, automation, operator
+    certified: "false"
+    description: Business Automation Operator can deploy RHPAM/RHDM environments in the form of KieApp objects.
+    containerImage: registry.redhat.io/rhpam-7-tech-preview/rhpam73-operator:1.0
+    createdAt: 2019-01-25T20:48:22Z
+    support: Red Hat, Inc.
+    tectonic-visibility: ocs
+    alm-examples: >-
+      [{"apiVersion":"app.kiegroup.org/v1","kind":"KieApp","metadata":{"name":"rhpam-trial"},"spec":{"environment":"rhpam-trial"}}]
+  labels:
+    operator-businessautomation: "true"
+spec:
+  displayName: Business Automation
+  description: Business Automation Operator can deploy RHPAM/RHDM environments in the form of KieApp objects.
+  keywords:
+    [
+      kieapp rhpam rhdm kie cloud pam bpm process case business automation decision operator,
```

Comment:
These should be a comma-separated list of strings. This can either be a yaml list (one line per keyword) or one line form (kinda like a JSON list).
Eg:
```
  keywords:
    - kieapp
    - rhpam
    - ...
```
or
```
  keywords: ["kieapp", "rhpam", ...]
```